phpBB/config/default/container/services_extensions.yml

```diff
@@ -0,0 +1,47 @@
+services:
+    ext.manager:
+        class: phpbb\extension\manager
+        arguments:
+            - '@service_container'
+            - '@dbal.conn'
+            - '@config'
+            - '@filesystem'
+            - '%tables.ext%'
+            - '%core.root_path%'
+            - '%core.php_ext%'
+            - '@cache'
+
+    ext.composer.installer:
+        class: phpbb\composer\installer
+        arguments:
+            - '%core.root_path%'
+            - '@filesystem'
+            - '@config'
+
+    ext.composer.manager:
+        class: phpbb\composer\extension_manager
+        arguments:
+            - '@ext.composer.installer'
+            - '@cache.driver'
+            - '@ext.manager'
+            - '@filesystem'
+            - phpbb-extension
```

Comment:
well I don't think it is an issue as they are constant and cannot change (phpbb-extension is composer dependent)